backend/apps/volontulo/factories.py

```diff
@@ -0,0 +1,28 @@
+import factory
+from django.contrib.auth import get_user_model
+
+from apps.volontulo.models import UserProfile
+
+
+User = get_user_model()
+
+
+class UserProfileFactory(factory.DjangoModelFactory):
+    class Meta:
+        model = UserProfile
+
+
+class UserFactory(factory.DjangoModelFactory):
+    class Meta:
+        model = User
+
+    username = factory.Sequence(lambda n: 'user{}@volontuloapp.org'.format(n))
```

Comment:
there's no reason why we should limit ourselves to `volontuloapp.org` domain

`fake.email(*args, **kwargs)`, see: https://faker.readthedocs.io/en/master/locales/pl_PL.html#faker-providers-internet